framework/auth/cas.py

```diff
@@ -0,0 +1,96 @@
+import furl
+import requests
+
+from lxml import etree
+
+
+class CasResponse:
+
+    def __init__(self):
+        self._authenticated = False
+        self._status = None
+        self._user = None
+        self._attributes = {}
+
+    @property
```

Comment:
What is the purpose of making `authenticated`, `status`, etc. properties instead of attributes?